.github/workflows/rails_tests.yml

```diff
@@ -0,0 +1,63 @@
+name: Rails Tests (non-blocking)
+
+on:
+  pull_request:
+    branches: [ master ]
+
+defaults:
+  run:
+    working-directory: ./foreman
+
+jobs:
+  build:
+    env:
+      PGHOST: 127.0.0.1
+      PGUSER: foreman
+      RAILS_ENV: test
+
+    runs-on: ubuntu-latest
+
+    services:
+      db:
+        image: quay.io/jomitsch/postgres-with-evr
```

Comment:
Nice work!
cc @ShimShtein now we could run rh_cloud test that uses Katello on ubuntu :+1: